ush/prepobs_oiqcbufr.sh

```diff
@@ -78,20 +78,14 @@ TIMEIT=${TIMEIT:-""}
 #$TIMEIT mpirun -genvall -n $LSB_DJOB_NUMPROC -machinefile $LSB_DJOB_HOSTFILE $OIQCX > outout 2> errfile
 
 SITE=${SITE:-""}
-sys_tp=${sys_tp:-$(getsystem.pl -tp)}
+sys_tp=${sys_tp:-$(getsystem -tp)}
```

Comment:
`getsystem -tp` is not valid on WC2. You need to use different options or none at all.